website/client/js/services/costumeServices.js

```diff
@@ -26,14 +26,36 @@
           potion: Content.hatchingPotions[info.potion].text()
         };
         animal[type == 'pet' ? 'egg' : 'mount'] = Content.eggs[info.egg].text()
-        return window.env.t(type+'Name', animal)
+        return window.env.t(type+'Name', animal);
       } else {
-        return type == 'pet' ? window.env.t(Content.specialPets[name]) : window.env.t(Content.specialMounts[name])
+        return type == 'pet' ? window.env.t(Content.specialPets[name]) : window.env.t(Content.specialMounts[name]);
       }
     }
-    
-    return {
-      formatAnimal: formatAnimal
+
+    function bgString(set, bg) {
+      var setString = window.env.t(set);
+      var setNumber = setString.match(/^[^:]+/)[0].toLowerCase();
+      var bgString = Content.backgrounds[set][bg].text(window.env.language.code);
+      return bgString + " - " + setNumber.charAt(0).toUpperCase() + setNumber.slice(1)
+    }
+
+    function formatBackground(background) {
+      var backgrounds = Content.backgrounds;
+      for (var set in backgrounds) {
+        for (var bg in backgrounds[set]) {
+          if (background === bg) {
+            return bgString(set, bg)
+          } else {
+            continue
```

Comment:
This is unnecessary; you're just going onto the next loop iteration, which you would be doing anyway.